projects/discussion-ui/src/lib/components/discuss-category/discuss-category.component.ts

```diff
@@ -55,8 +55,13 @@ export class DiscussCategoryComponent implements OnInit, OnDestroy {
       if (_.get(params, 'cid')) {
         this.navigateToDiscussionPage(_.get(params, 'cid'));
       } else {
-        this.categories = [];
-        this.fetchAllAvailableCategories(this.forumIds);
+        if (this.forumIds.length) {
+          this.categories = [];
+          this.fetchAllAvailableCategories(this.forumIds);
+        } else {
+          this.categories = [];
```

Comment:
This is also done in else block.....can we not take that out if,else condition and write in single place